docs/elasticsearch-specification.asciidoc

```diff
@@ -457,20 +460,20 @@ spec:
       maxSurge: 3
       maxUnavailable: 1
 ----
-`maxSurge` refers to the number of Pods that can be scheduled above the total number of Pods in the currently applied specification. This setting might be useful when the new specification has similar number of Pods, but different node sets (for example, when migrating Pods to a new node set). It will prevent the operator from creating all new Pods before removing the old ones thus preventing having (temporarily) as many as two times more pods than the specification defines.
+`maxSurge`: Refers to the number of Pods that can be scheduled out of the total number of Pods in the currently applied specification. This setting is useful when the new specification has similar number of Pods, but different node sets. For example, when migrating Pods to a new node set. This setting prevents the operator from creating all new Pods before removing the old ones, by temporarily avoiding two times more Pods than the ones defined in the specification.
```

Comment:
The original text was a bit confusing so the meaning got lost in the re-write. I would try to explain `MaxSurge` as follows.  
```suggestion
`maxSurge`: Refers to the number of extra Pods that can be temporarily scheduled exceeding the number of Pods  defined in the specification. This setting is useful for controlling the resource usage of the Kubernetes cluster when new Pods need to be spun up to replace existing Pods. For example, when applying a configuration change to a Node Set, ECK creates a clone of the Node Set with the desired configuration and shuts down the old Pods. Without the `MaxSurge` restriction, the cluster would temporarily contain twice as many Pods as usual while this operation is in progress.
```